tests/std/tests/P0645R10_text_formatting_formatting/test.cpp

```diff
@@ -8,252 +8,359 @@
+    assert(output_string == TYPED_LITERAL(charT, "}"));
 
     output_string.clear();
-    vformat_to(back_insert_iterator(output_string), locale::classic(), "}}}}", make_format_args());
-    assert(output_string == "}}");
+    vformat_to(back_insert_iterator(output_string), locale::classic(), TYPED_LITERAL(charT, "}}}}"),
+        make_testing_format_args<charT>());
+    assert(output_string == TYPED_LITERAL(charT, "}}"));
 
     output_string.clear();
-    vformat_to(back_insert_iterator(output_string), locale::classic(), "x}}", make_format_args());
-    assert(output_string == "x}");
+    vformat_to(back_insert_iterator(output_string), locale::classic(), TYPED_LITERAL(charT, "x}}"),
+        make_testing_format_args<charT>());
+    assert(output_string == TYPED_LITERAL(charT, "x}"));
 
     output_string.clear();
-    vformat_to(back_insert_iterator(output_string), locale::classic(), "}} }}", make_format_args());
-    assert(output_string == "} }");
+    vformat_to(back_insert_iterator(output_string), locale::classic(), TYPED_LITERAL(charT, "}} }}"),
+        make_testing_format_args<charT>());
+    assert(output_string == TYPED_LITERAL(charT, "} }"));
 
     output_string.clear();
-    vformat_to(back_insert_iterator(output_string), locale::classic(), "x}}x", make_format_args());
-    assert(output_string == "x}x");
+    vformat_to(back_insert_iterator(output_string), locale::classic(), TYPED_LITERAL(charT, "x}}x"),
+        make_testing_format_args<charT>());
+    assert(output_string == TYPED_LITERAL(charT, "x}x"));
 
     output_string.clear();
-    vformat_to(back_insert_iterator(output_string), locale::classic(), "}}x", make_format_args());
-    assert(output_string == "}x");
+    vformat_to(back_insert_iterator(output_string), locale::classic(), TYPED_LITERAL(charT, "}}x"),
+        make_testing_format_args<charT>());
+    assert(output_string == TYPED_LITERAL(charT, "}x"));
+}
 
-    output_string.clear();
-    vformat_to(back_insert_iterator(output_string), locale::classic(), "{}", make_format_args("f"));
-    assert(output_string == "f");
+template <class charT>
+void test_simple_replacement_field() {
+    basic_string<charT> output_string;
 
-    output_string.clear();
-    vformat_to(back_insert_iterator(output_string), locale::classic(), "{0} {0}", make_format_args("f"));
-    assert(output_string == "f f");
+    vformat_to(back_insert_iterator(output_string), locale::classic(), TYPED_LITERAL(charT, "{}"),
+        make_testing_format_args<charT>(TYPED_LITERAL(charT, "f")));
+    assert(output_string == TYPED_LITERAL(charT, "f"));
 
     // Test string_view
     output_string.clear();
-    vformat_to(back_insert_iterator(output_string), locale::classic(), "{}", make_format_args("f"sv));
-    assert(output_string == "f");
+    vformat_to(back_insert_iterator(output_string), locale::classic(), TYPED_LITERAL(charT, "{}"),
+        make_testing_format_args<charT>(basic_string_view<charT>(TYPED_LITERAL(charT, "f"))));

```

Comment:
Similar comment for using braces to construct this `basic_string_view<charT>` temporary.